pkg/agent/controller/ipseccertificate/ipsec_certificate_controller_test.go

```diff
@@ -353,6 +353,11 @@ func signCSR(t *testing.T, controller *fakeController,
 	assert.Empty(t, remain)
 	req, err := x509.ParseCertificateRequest(block.Bytes)
 	assert.NoError(t, err)
+
+	// Wait one second as the fake clientset doesn't support watching with specific resourceVersion.
+	// The update event may be missed by the watcher used in csrutil.WaitForCertificate()
```

Comment:
```suggestion
	// Wait one second as the fake clientset doesn't support watching with specific resourceVersion.
	// Otherwise the update event would be missed by the watcher used in csrutil.WaitForCertificate()
	// if it happens to be generated in-between the List and Watch calls.
```

Reply:
updated. thanks!